src/completions/FileSystem.ts

```diff
@@ -0,0 +1,78 @@
+import * as Completions from "@src/completions"
+import * as Messaging from "@src/lib/messaging"
+import * as config from "@src/lib/config"
+
+class FileSystemCompletionOption extends Completions.CompletionOptionHTML
+    implements Completions.CompletionOptionFuse {
+    public fuseKeys = []
+
+    constructor(public value: string) {
+        super()
+        this.fuseKeys = [value]
+        this.html = html`<tr class="FileSystemCompletionOption option">
+            <td class="value">${value}</td>
+        </tr>`
+    }
+}
+
+export class FileSystemCompletionSource extends Completions.CompletionSourceFuse {
+    public options: FileSystemCompletionOption[]
+
+    constructor(private _parent) {
+        super(["saveas"], "FileSystemCompletionSource", "FileSystem")
```

Comment:
Could you add `source` here before I forget?

Reply:
Done.